lib/aeuis.js

```diff
@@ -0,0 +1,39 @@
+var Client = require("./client");
+var AEUIS_PATH = "aeuis";
+
+function Aeuis(){
```

Comment:
Does this need to be a function? Can it just be an object?

Reply:
This was done to keep in the practice of how the SDK was written so there no varying styles. But if the SDK is every refactored a class would be a better way.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes

Reply:
Don't think classes would be better per say. The sdk uses objects and functions as top level which is confusing. See #11